_sass/_layout.scss

```diff
@@ -18,41 +18,98 @@ footer {
   }
 }
 
+// hide open button for mobile versions (while in desktop mode)
+.openbtn {

```

Comment:
Since we have Sass setup already, in the future I think it would help to separate our partials css just for readability